src/releng_docs/services_shipit.rst

```diff
@@ -56,4 +56,61 @@ It does the following tasks on every run:
 - Report the full analysis to shipit dashboard, so it can be displayed on shipit frontend.
 
 
+``shipit v1 and v2 interaction``
+--------------------------------
+
+Initially, shipit-v2 will compliment v1 by adding features, as opposed to replacing. The following is a implementation strategy for how that will work.
+
+**ship-it v1 will solely be responsible for submitting and kicking off a new release**
+
+.. code-block:: bash
+
+    % curl -X POST -d "product=firefox&version=99.0" https://ship-it.mozilla.org/submit_release.html
+
+releaserunner_ continues to behave as normal by polling shipit v1 for new releases and, when found, runs release_sanity_, creates taskcluster_graph1_, and notifies release-drivers of new release.
+In addition, releaserunner will tell shipit-v2 about the new release and ask shipit-v2 to create a pipeline:
+
+.. code-block:: python
+
+    r = requests.post("https://mozilla-releng.net/shipit_pipeline", data={uid='foo', pipeline={}})
```

Comment:
The URL for pipeline API service is going to be ``https://pipeline.shipit.mozilla-releng.net``.

This follows the naming convention of <service>.<domain> where <domain> is ``shipit.mozilla-releng.net`` (we might change this later if needed) and <service> is the name of the application in ``mozilla-releng/services`` without *family* name. eg: ``shipit_something`` has a *family* is ``shipit`` and name is ``something``. Family names are used to group development environments together, eg. provide one docker image for all of the *family*. Hope this clears things a bit.